eva-lib/src/test/java/uk/ac/ebi/eva/lib/metadata/VariantSourceEvaProDBAdaptorTest.java

```diff
@@ -23,6 +23,7 @@
 import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
 import org.springframework.test.context.junit4.SpringRunner;
 import uk.ac.ebi.eva.commons.core.models.stats.VariantSourceStats;
+import uk.ac.ebi.eva.lib.metadata.eva.VariantSourceEvaProDBAdaptor;
```

Comment:
This test should be moved to the same package that the class that is testing: uk.ac.ebi.eva.lib.metadata.eva